ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="tile component for product listing page"
+    jcr:primaryType="cq:Component"
+    jcr:title="Tile"
```

Comment:
@ReenaOmana : Component name will be Product Listing Tile

Reply:
Changes done